assets/src/application/ui/layout/espressoTable/utils.ts

```diff
@@ -1,20 +1,6 @@
-import React, { Children } from 'react';
+import { Children } from 'react';
 import { castArray, filter, first, isArray, isEmpty, isFunction, last, reject } from 'lodash';
 
-import { DisplayDates } from '@edtrInterfaces/shared/types';
-
-export const displayDatesFilter = (displayDates) => (cell) => {
-	if (displayDates === DisplayDates.start && cell.key === 'end') {
-		return null;
-	}
-
-	if (displayDates === DisplayDates.end && cell.key === 'start') {
-		return null;
-	}
-
-	return cell;
-};
-
```

Comment:
`/application` should avoid imports from `/domain`. I moved it to `/domain/shared`

Reply:
**HUGE** 👍 

domain (use cases) layer can have external dependencies
but other layers can NOT have dependencies on anything in the domain layer